specification/machinelearningservices/resource-manager/Microsoft.MachineLearningServices/preview/2023-04-01-preview/registries.json

```diff
@@ -579,7 +579,7 @@
           "$ref": "#/definitions/PrivateEndpointResource",
           "x-nullable": true
         },
-        "registryPrivateLinkServiceConnectionState": {
```

Comment:
This change is not a breaking change -- rather a fix of a breaking change that was accidentally added in the April API versions. There was a conflict between aliased contracts for private endpoint connections between registries and machinelearningservice swagger documents, which was fixed by adding "registry" to the names of these contracts, but someone added these names to the properties as well which was not necessary and causes our swagger to no longer match the contracts for that API version.